api-reference/10 UI Components/dxChat/1 Configuration/user.md

```diff
@@ -5,7 +5,11 @@ default: { id: new Guid().toString() }
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies a current chat user (to the right).
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+This property has the following limitations for the current user:
```

Comment:
I wouldn't call those items "limitations". A limitation would not allow me to do something. But we seem to allow developers to do what they need to do. Even more, if they didn't specify what's needed, we generate it for them. 

```suggestion
This property value, or its part, may be specified automatically:
```